src/main/java/org/ibp/api/rest/study/StudyEntryResource.java

```diff
@@ -51,17 +54,30 @@
 
 	}
 
-	@ApiOperation(value = "Create germplasm entries in study based on the specified germplasm list",
-		notes = "Create germplasm entries in study based on the specified germplasm list")
+	@ApiOperation(value = "Create germplasm entries in study based on the specified germplasm ids",
+		notes = "Create germplasm entries in study based on the specified germplasm ids")
 	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/entries/generation", method = RequestMethod.POST)
 	@ResponseBody
 	public ResponseEntity<List<StudyEntryDto>> createStudyEntries(final @PathVariable String cropname,
-		@PathVariable final String programUUID,
-		@PathVariable final Integer studyId, @RequestBody final GermplasmEntryRequestDto germplasmEntryRequestDto) {
+		@PathVariable final String programUUID,	@PathVariable final Integer studyId,
+		@ApiParam("Study Entry template for batch generation. SearchComposite is a list of gids")
+		@RequestBody final StudyEntryGeneratorRequestDto studyEntryGeneratorRequestDto) {
 		return new ResponseEntity<>(
-			this.studyEntryService.createStudyEntries(studyId, germplasmEntryRequestDto.getGermplasmListId()),
+			this.studyEntryService.createStudyEntries(studyId, studyEntryGeneratorRequestDto),
 			HttpStatus.OK);
+	}
 
+	@ApiOperation(value = "Create germplasm entries in study based on the specified germplasm list",
+		notes = "Create germplasm entries in study based on the specified germplasm list")
+	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/entries/generation-list", method = RequestMethod.POST)
```

Comment:
@mderamos 
I think it make more sense to use  
`POST /{cropname}/programs/{programUUID}/studies/{studyId}/entries/generation`
for this service and rename the other one